### PR TITLE
docs: add configuration file crud instructions

### PR DESCRIPTION
**Pull request by plantaz**
## Description of change

Add instructions on how to add, edit and remove configuration files

## Issues resolved by this PR
- [x] [DVN-10223]
- [x] [DVN-10224]
- [x] [DVN-10225]

<details>
  <summary>More info to help validate your PR: links, images, videos, ...</summary>
  
Before
image
After
image

</details>

[DVN-10223]: https://devopness.atlassian.net/browse/DVN-10223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10224]: https://devopness.atlassian.net/browse/DVN-10224?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10225]: https://devopness.atlassian.net/browse/DVN-10225?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/configuration-files/add-configuration-file.md [hunk 1]
```diff
@@ -12,3 +12,12 @@ links:
     related:
     featured:
 ---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application where you want to add a configuration file and click on`DETAILS`
```
Comment (jfoliveira) [inline docs/configuration-files/add-configuration-file.md:19]: There is a space missing after `on`. In other topics we usually favor just saying `click <thing to be clicked>`. The more straightforward the instructions are less friction we're gonna have correcting English grammar on our docs.

File: docs/configuration-files/edit-configuration-file.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the configuration file you want to edit and click on`DETAILS`
```
Comment (jfoliveira) [inline docs/configuration-files/edit-configuration-file.md:17]: Likewise, `on` preposition here is not adhering to our standards.

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/configuration-files/add-configuration-file.md (hunk 1 above)
Comment (plantaz) [reply docs/configuration-files/add-configuration-file.md]: I agree, just removed the 'on'.

File: docs/configuration-files/edit-configuration-file.md (hunk 2 above)
Comment (plantaz) [reply docs/configuration-files/edit-configuration-file.md]: I agree, just removed the 'on'.

File: docs/configuration-files/edit-configuration-file.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the configuration file you want to edit and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
+6. Use the drop-down menu to chose `CONFIGURATION FILES`
+8. In the list of configuration files find the configuration file you want to edit and click `DETAILS`
+9. On the upper-right corner of the key details click `...`
+10. Use the drop-down menu to chose `EDIT`
```
Comment (jfoliveira) [inline docs/configuration-files/edit-configuration-file.md:22]: ~~chose~~
choose

File: docs/configuration-files/add-configuration-file.md [hunk 4]
```diff
@@ -12,3 +12,12 @@ links:
     related:
     featured:
 ---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application where you want to add a configuration file and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
+1. Use the drop-down menu to chose `CONFIGURATION FILE`
```
Comment (jfoliveira) [inline docs/configuration-files/add-configuration-file.md:21]: ~~chose~~
choose

File: docs/configuration-files/remove-configuration-file.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the pipeline you want to manage and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
+6. Use the drop-down menu to chose `CONFIGURATION FILES`
+8. In the list of pipelines find the pipeline you want to remove and click `DETAILS`
+9. On the upper-right corner of the configuration file details click `...`
+10. Use the drop-down menu to chose `REMOVE`
```
Comment (jfoliveira) [inline docs/configuration-files/remove-configuration-file.md:24]: ~~chose~~
choose

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/configuration-files/edit-configuration-file.md (hunk 3 above)
Comment (plantaz) [reply docs/configuration-files/edit-configuration-file.md]: Fixed

File: docs/configuration-files/add-configuration-file.md (hunk 4 above)
Comment (plantaz) [reply docs/configuration-files/add-configuration-file.md]: Fixed

File: docs/configuration-files/remove-configuration-file.md (hunk 5 above)
Comment (plantaz) [reply docs/configuration-files/remove-configuration-file.md]: Fixed

File: docs/configuration-files/add-configuration-file.md [hunk 6]
```diff
@@ -12,3 +12,12 @@ links:
     related:
     featured:
 ---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application where you want to add a configuration file and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
```
Comment (thlmenezes) [inline docs/configuration-files/add-configuration-file.md:20]:
```suggestion
1. In the list of applications find the application where you want to add a configuration file and click `DETAILS`
1. On the upper-right corner of the applications details view click `SETTINGS`
```

Cleaner option for ordered list count, avoiding to worry on item numbering

File: docs/configuration-files/add-configuration-file.md [hunk 7]
```diff
@@ -12,3 +12,12 @@ links:
...
+1. Use the drop-down menu to choose `CONFIGURATION FILE`
+1. On the right corner of the list click `ADD CONFIGURATION FILE`
+2. Follow the instructions in the add configuration file step form then click `CONFIRM`
```
Comment (thlmenezes) [inline docs/configuration-files/add-configuration-file.md:23]:
```suggestion
1. Follow the instructions in the add configuration file step form then click `CONFIRM`
```

File: docs/configuration-files/edit-configuration-file.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the configuration file you want to edit and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
+6. Use the drop-down menu to choose `CONFIGURATION FILES`
+8. In the list of configuration files find the configuration file you want to edit and click `DETAILS`
+9. On the upper-right corner of the key details click `...`
+10. Use the drop-down menu to choose `EDIT`
+11. Follow the instructions in the configuration file edit form then click `REMOVE`
```
Comment (thlmenezes) [inline docs/configuration-files/edit-configuration-file.md:23]:
```suggestion
1. On Devopness, navigate to a project then select an environment
1. Find the `Applications` card
1. Click `View` in the `Applications` card to see a list of existing `Applications`
1. In the list of applications find the application with the configuration file you want to edit and click `DETAILS`
1. On the upper-right corner of the applications details view click `SETTINGS`
1. Use the drop-down menu to choose `CONFIGURATION FILES`
1. In the list of configuration files find the configuration file you want to edit and click `DETAILS`
1. On the upper-right corner of the key details click `...`
1. Use the drop-down menu to choose `EDIT`
1. Follow the instructions in the configuration file edit form then click `REMOVE`
```

File: docs/configuration-files/remove-configuration-file.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the pipeline you want to manage and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
+6. Use the drop-down menu to choose `CONFIGURATION FILES`
+8. In the list of pipelines find the pipeline you want to remove and click `DETAILS`
+9. On the upper-right corner of the configuration file details click `...`
+10. Use the drop-down menu to choose `REMOVE`
+11. Follow the instructions in the delete form then click `REMOVE`
```
Comment (thlmenezes) [inline docs/configuration-files/remove-configuration-file.md:25]:
```suggestion
1. On Devopness, navigate to a project then select an environment
1. Find the `Applications` card
1. Click `View` in the `Applications` card to see a list of existing `Applications`
1. In the list of applications find the application with the pipeline you want to manage and click `DETAILS`
1. On the upper-right corner of the applications details view click `SETTINGS`
1. Use the drop-down menu to choose `CONFIGURATION FILES`
1. In the list of pipelines find the pipeline you want to remove and click `DETAILS`
1. On the upper-right corner of the configuration file details click `...`
1. Use the drop-down menu to choose `REMOVE`
1. Follow the instructions in the delete form then click `REMOVE`
```

File: docs/configuration-files/edit-configuration-file.md (hunk 8 above)
Comment (thlmenezes) [inline docs/configuration-files/edit-configuration-file.md:23]:
```suggestion
11. Follow the instructions in the configuration file edit form then click `CONFIRM`
```

image

https://app.devopness.com/projects/2/environments/2/applications/102/files/124/edit

**Review by thlmenezes — CHANGES_REQUESTED** [review]

**Comment by thlmenezes** [comment]
@plantaz, could you connect this PR with the Jira Issues related to it? Creating a section in your PR description following [Slab - Devopness: Jira Issue list](https://devopness.slab.com/posts/code-review-guidelines-and-policies-hbrqcche#h103u-jira-issues-list) will connect them.

[DVN-10223] and its children

**Review by thlmenezes — APPROVED** [review]

**Review by thlmenezes — APPROVED** [review]

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**